src/components/common/CRCellButton.js

```diff
@@ -0,0 +1,34 @@
+"use-strict";
+import React, { Component } from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+
+class CRCellButton extends Component {
```

Comment:
choose a descriptive name, what doe `CRCellButton` means?

Reply:
Conference Room Cell Button -> is the button that belongs to the cells of My Meetings and Fast Meetings. and Conference Room because of the app name

Reply:
can we assign a more declarative name?

Reply:
yes, just let me think a better one, I'm bad with variables and components names 😢

Reply:
check it out